backend/config/curation-api.yml

```diff
@@ -46,20 +50,21 @@ paths:
 
   /v1/collections:
     get:
-      summary: Fetch Collections metadata.
-      description: >-
-        * When the visibility parameter is unspecified or set to `PUBLIC`, a list of all public collections is returned.
-        The Authorization header is not required. If a Collection in the list has been deleted, then it is annotated
-        with tombstone set to True.
+      summary: List Collections' metadata.
+      description: |
+        For each Collection in the list, the datasets for that Collection include a subset of dataset metadata as a 
+        preview.To retrieve full dataset metadata for a specific Collection, GET /v1/collections/{collection_id} must 
```

Comment:
```suggestion
        preview. To retrieve full dataset metadata for a specific Collection, GET /v1/collections/{collection_id} must 
```

Reply:
also `dataset` -> `Dataset` in this section please. There are a couple instances above this line, too.